src/SB/Core/gc/iModel.cpp

```diff
@@ -49,15 +76,65 @@ void iModelUnload(RpAtomic* userdata)
     if ((RpClump*)userdata != 0)
     {
         RwFrameGetRoot((RwFrame*)userdata);
+
+        userdata = 0;
+    }
+    else
+    {
+        RwFrameDestroyHierarchy(frame);
+
         RwFrameDestroyHierarchy(frame);
         userdata = 0;
+
     }
     if (clump != 0)
     {
         RpClumpDestroy(clump);
     }
 }
 
+
+RpAtomic* NextAtomicCallback(RpAtomic* atomic, void* data)
+{
+    // What?
+
```

Comment:
Could you clarify this comment? Is there something here that you felt might be particularly confusing for future contributors?